lib/handler.js

```diff
@@ -1,11 +1,14 @@
 "use strict" ;
 
+mongoose = require('mongoose');
+Result = require('./result.js')
+
 module.exports = class Handler {
   /* Initializes cumulative results structure for storage at finish time, and initializes connection to MongoDB
     */
 
   /* TODO: Placeholder mongodb url */
-  mongo_url = "";
+  mongo_url = "mongodb://127.0.0.1:27017";
```

Comment:
Is this `handler` designed to be the MongoDB-only handler? Otherwise we should figure out how to migrate the MongoDB connection stuff into the various sources

Reply:
It's not only a MongoDB handler--it also handles aggregation.

Reply:
I more meant "is this handler designed for using MongoDB only, as opposed to being data source agnostic?"

Reply:
Ah, gotcha. Yeah, I think we wanted it to be data source agnostic. In which case you're right, we should figure out how to migrate the MongoDB connection stuff, though I figure that will also happen when we implement a config file.

Reply:
That being said, there may need to be specific implementations of different data sources in the handler, rather than a single method that will work with several (example: it already interfaces directly with Mongoose for now).